agreement_dynamic/demo/demo.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <!-- Agreement -->
+    <record id="demo_agreement_1" model="agreement">
```

Comment:
In this demo record I wanted also to initialize the reference field. How do I do that?

Reply:
Try either:

```
<field name="resource_ref" ref="hr.employee_mit" />
```

Or:

```
<field name="model">hr.employee</field>
<field name="res_id" ref="hr.employee_mit" />
```

I'm pretty sure that either of these two will work.

Remove the demo record again before updating your model though - demo records are created, but not updated, so you'd be waiting for white smoke forever.

Reply:
(model = module)